.travis.yml

```diff
@@ -11,8 +11,12 @@ env:
           use_coveralls=false
         - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
           testflags="--test-type integ_slow --pytest-args='-x'"
-        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev-nm-1.22-git
-          testflags="--test-type integ --pytest-args='-x'"
+        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
+          testflags="--test-type integ --pytest-args='-x'
+              --copr networkmanager/NetworkManager-1.22-git"
+        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
+          testflags="--test-type integ --pytest-args='-x'
+              --copr networkmanager/NetworkManager-1.22-master"
```

Comment:
This should be  `--copr networkmanager/NetworkManager-master` because `networkmanager/NetworkManager-1.22-master` does not exist. Thanks!

Reply:
Thanks.